packages/opentelemetry-core/src/utils/environment.ts

```diff
@@ -156,26 +170,13 @@ function setLogLevelFromEnv(
   values: RAW_ENVIRONMENT
 ) {
   const value = values[key];
-  switch (typeof value === 'string' ? value.toUpperCase() : value) {
-    case 'DEBUG':
-      environment[key] = LogLevel.DEBUG;
-      break;
-
-    case 'INFO':
-      environment[key] = LogLevel.INFO;
-      break;
-
-    case 'WARN':
-      environment[key] = LogLevel.WARN;
-      break;
-
-    case 'ERROR':
-      environment[key] = LogLevel.ERROR;
-      break;
-
-    default:
-      // do nothing
-      break;
+  if (typeof value === 'string') {
+    const theLevel = logLevelMap[value.toUpperCase()];
+    if (theLevel != undefined) {
+      environment[key] = theLevel;
+    }
+  } else if (typeof value === 'number') {
+    environment[key] = value;
```

Comment:
Question: Do we want to support numeric values at this point in time?

Reply:
No

Reply:
Removed.